File: backend/src/main/java/ua/meetuply/backend/model/Meetup.java
package ua.meetuply.backend.model;

import lombok.*;

import java.time.LocalDateTime;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Meetup {
    private Integer meetupId;
    private String meetupPlace;
    private String meetupTitle;
    private String meetupDescription;
    private int meetupRegisteredAttendees;
    private int meetupMinAttendees;
    private int meetupMaxAttendees;
    private LocalDateTime meetupStartDateTime;
    private LocalDateTime meetupFinishDateTime;
    private Integer stateId;
    private Integer speakerId;
    private String speakerFirstName;
    private String speakerLastName;
    private Float rating;
    private String speakerPhoto;
    private String language;
    private List<Integer> topics;
}

File: frontend/src/main/web/src/app/_models/meetup.ts
import { Language } from "./language";
import { Topic } from "./topic";

export class Meetup {
  meetupId: number;
  meetupPlace: string;
  meetupTitle: string;
  meetupDescription: string;
  meetupRegisteredAttendees: number;
  meetupMinAttendees: number;
  meetupMaxAttendees: number;
  meetupStartDateTime: Date;
  meetupFinishDateTime: Date;
  stateId: number;
  speakerId: number;

  language: string;
  topics: number[];
}
